app/code/Magento/Catalog/Controller/Adminhtml/Product/Attribute/Save.php

```diff
@@ -192,11 +192,12 @@ public function execute()
                     $data['frontend_input']
                 );
             }
-
             $data += ['is_filterable' => 0, 'is_filterable_in_search' => 0, 'apply_to' => []];
 
-            if (is_null($model->getIsUserDefined()) || $model->getIsUserDefined() != 0) {
-                $data['backend_type'] = $model->getBackendTypeByInput($data['frontend_input']);
+            if ($model->getIsUserDefined()) {
+                if (empty($model->getIsUserDefined())) {
```

Comment:
For which value of `$model->getIsUserDefined()` will it ever pass into?

Reply:
Oh, sorry bad commit ,